src/components/Dashboard/Capacity/CapacityItem.js

```diff
@@ -27,43 +21,38 @@ export const CapacityItem = ({ id, title, used, total, unit, formatValue, Loadin
     const totalFormatted = formatValue(total, unit);
     const usedFormatted = formatValue(used, totalFormatted.unit);
 
-    unit = totalFormatted.unit; // eslint-disable-line prefer-destructuring
-    total = totalFormatted.value;
-    used = usedFormatted.value;
+    const formattedUnit = totalFormatted.unit; // eslint-disable-line prefer-destructuring
+    const formattedTotal = totalFormatted.value;
+    const formattedUsed = usedFormatted.value;
 
-    const available = total - used;
-    columns = [[USED, used], [AVAILABLE, available]];
+    const available = formattedTotal - formattedUsed;
+    data = [{ x: USED, y: formattedUsed }, { x: AVAILABLE, y: available }];
 
-    const percentageUsed = total > 0 ? Math.round((100 * used) / total) : 0;
+    const percentageUsed = total > 0 ? Math.round((100 * formattedUsed) / formattedTotal) : 0;
     const availableNice = Number.isInteger(available) ? available : available.toFixed(1);
-    const totalNice = Number.isInteger(total) ? total : total.toFixed(1);
+    const totalNice = Number.isInteger(formattedTotal) ? formattedTotal : formattedTotal.toFixed(1);
 
-    description = `${availableNice} ${unit} available out of ${totalNice} ${unit}`;
-    tooltip = { contents: pfGetUtilizationDonutTooltipContents };
+    description = `${availableNice} ${formattedUnit} available out of ${totalNice} ${formattedUnit}`;
     primaryTitle = `${percentageUsed.toString()}%`;
     secondaryTitle = USED;
   }
-  // TODO(mlibra): align text font
   return (
     <div className="kubevirt-capacity__item">
       <h3 className="kubevirt-capacity__item-title">{title}</h3>
       <h6>{description}</h6>
-      <div>
-        <DonutChart
-          id={prefixedId('donut-chart', id)}
-          size={{ width: 150, height: 150 }}
-          data={{
-            unload: true,
-            columns,
-            order: null,
-          }}
-          unloadBeforeLoad
-          tooltip={tooltip}
-          title={{
-            primary: primaryTitle,
-            secondary: secondaryTitle,
-          }}
-        />
+      <div id={prefixedId('donut-chart', id)} className="kubevirt-capacity__graph">
+        <svg viewBox="0 0 200 200">
```

Comment:
can we make variables out of the height and the width?

Reply:
created constants out of width and height